### ISSUE TITLE
UTF-8 message body breaks mail parser

### ISSUE THREAD

**Issue by sj26** [mikel/mail#1235]
The following seems to break the Mail parser:

```ruby
body = <<EOM
From: Someone <someone@example.com>
To: Someone Else <someone-else@example.com>
Subject: Something

UTF-8の文字列
EOM

body.encoding
# => #<Encoding:UTF-8>

message = Mail.new(body)
# => #<Mail::Message:70136147117000, Multipart: false, Headers: <From: Someone <someone@example.com>
# To: Someone Else <someone-else@example.com>
# Subject: Something
# 
# UTF-8の文字列>>

message.to
# => nil

message.subject
# => nil

message.body.to_s
# => ""

message.from
#  => "Someone <someone@example.com>\nTo: Someone Else <someone-else@example.com>\nSubject: Something\n\nUTF-8の文字列"
```

Removing the utf-8 body makes it work.

Should mail be more lenient here? Considering the string is encoded as utf-8, perhaps it should parse message bodies of utf-8 characters and make sure they're suitably encoded for transfer?

**Comment by sj26** [comment]
Maybe related to #634

**Comment by jeremy** [comment]
This is due to the source email using LF (\n) line separators rather than CRLF (\r\n). See #1167 and #1168.

**Comment by sj26** [comment]
Great, thanks @jeremy — looks like it's fixed again in 2.7.1.rc1.
